mono/mini/mini.h

```diff
@@ -1217,6 +1217,10 @@ typedef struct {
 	int active_jit_methods;
 
 	gpointer interp_context;
+
+#if defined(TARGET_WIN32)
+	MonoContext stack_restore_ctx;
```

Comment:
This should be HOST_WIN32.

Reply:
When do we use which? All of the usages of this in exceptions-amd64.c are already wrapped in `#ifdef TARGET_WIN32`?

Reply:
HOST_X and TARGET_X can be different when building for example a mono cross compiler. The HOST is where the runtime will execute while the TARGET is related to codegen (together with CPU arch defines etc).

So if you build a runtime that will run and generate code on Windows both HOST_WIN32 and TARGET_WIN32 will be set. If you are building a mono cross compiler that will run on Windows but generate code for a different platform they will be different.

I believe that the mono Windows codebase has not been very consistent in the past so there are probably several areas where these defines have been mixed and switched. That is something to look over in the future and I believe we will try to build more mono cross compilers targeting other platforms using the Windows build as well.

Reply:
Yes, in this case I use this field only for generating code for amd64 on Windows 64. I would expect to wrap in `TARGET_WIN32`?